rnb-cli/src/tools/rnbv.js

```diff
@@ -7,16 +7,19 @@ const semver = require("semver")
 
 const configMap = new Map([
   ["androidPath", "./android/app/build.gradle"],
-  ["iosPath", "./ios/ReactNativeBase/Info.plist"],
+  ["iosPath", "./ios/ReactNativeBase.xcodeproj/project.pbxproj"],
```

Comment:
We only have one Info.plist, changing version and build numbers here will be applied to all schemes, that's why we now use `project.pbxproj` to edit MARKETING_VERSION and CURRENT_PROJECT_VERSION per scheme.